src/website/views/getting-started/whats-new.pug

```diff
@@ -6,7 +6,23 @@ script(type='module' src='../../js/ce/ux-chi-ce/ux-chi-ce.esm.js')
 script(nomodule='' src='../../js/ce/ux-chi-ce/ux-chi-ce.js')
 
 .changelog__version.-py--1
-  h2.-text--h2.-text--bold.-mb--1 <a href="https://github.com/CenturyLinkCloud/ux-chi/releases/tag/v3.13.0" target="_blank">Chi v3.13.0</a>
+  h2.-text--h2.-text--bold.-mb--1 <a href="https://github.com/CenturyLink/Chi/releases/tag/v3.14.0" target="_blank">Chi v3.14.0</a>
+  .-text--grey.-text--md March 25, 2021
+  .-pt--1
+    h3.-text--h3.-text--bold Components
+    h4.-text--h4.-text--bold Added
+    ul.-text.-pl--2
+      li Added: Search input component to provide users with the ability input words or phrases and locate relevant content.
+      li Added: Accordion now supports the class <code>-truncated</code> to truncate long accordion titles.
+    h3.-text--h3.-text--bold Documentation
+    h4.-text--h4.-text--bold Changed
+    ul.-text.-pl--2
+      li Changed: Chi GitHub links have been updated to reflect our new repository URL.
+
+.chi-divider.-my--4
+
+.changelog__version.-py--1
+  h2.-text--h2.-text--bold.-mb--1 <a href="https://github.com/CenturyLink/Chi/releases/tag/v3.13.0" target="_blank">Chi v3.13.0</a>
```

Comment:
@mattnickles @jllr Are we going to update release tag URLs of previous versions with URLs pointing to the new repository?